docs/21-distributing-data.md

```diff
@@ -27,8 +27,11 @@ results <- foreach(chunk = iter(chunks)) %dopar% {
 
 ## Pre-loading Data Into The Cluster
 
-Some workloads may require data pre-loaded into the cluster as soon as the cluster is provisioned. doAzureParallel supports this with the concept of a *resource file* - a file that is automatically downloaded to each node of the cluster after the cluster is created. Note: Creating containers's default setting is private. It must be changed to public if user does not want to use SAS tokens.
+Some workloads may require data pre-loaded into the cluster as soon as the cluster is provisioned. doAzureParallel supports this with the concept of a *resource file* - a file that is automatically downloaded to each node of the cluster after the cluster is created.
 
+**NOTE** The default setting for storage containers is _private_. You can either use a SAS (link to sas docs) to access the resources or [make the container public using the Azure Portal](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-manage-access-to-resources).
```

Comment:
:) "link to SAS docs".

"Should be you can either use a [SAS](../samples/resource_files/sas_resource_files_example.R) to access"

Please verify I got the relative link right.